.travis.yml

```diff
@@ -1,7 +1,16 @@
-before_script:
-  - sh -e /etc/init.d/xvfb start
-  - export DISPLAY=:99.0
-bundler_args: --without development production --quiet
+language: ruby
+
+cache:
+  bundler: true
+
+rvm:
+  - 2.3.1
+  - 2.2.5
+  - 2.1.10
```

Comment:
@krzysiek1507 currently yes, but master will switch to `rails 5` soon and we will drop ruby `2.1.x` support